.github/workflows/contracts.yaml

```diff
@@ -196,20 +215,80 @@ jobs:
           yarn run hardhat --network ${{ github.event.inputs.environment }} \
             etherscan-verify --license GPL-3.0 --force-license
 
+  # This job is responsible for publishing packages from `dapp-development`
+  # branch, which are slightly modified to help with the process of testing some
+  # features on the Threshold Token dApp. The job starts only if workflow gets
+  # triggered by the `workflow_dispatch` event on the branch `dapp-development`.
+  contracts-dapp-development-deployment-testnet:
```

Comment:
I'm wondering whether common parts of `contracts-deployment-testnet` and `contracts-dapp-development-deployment-testnet` could be extracted to a reusable workflow or action. But this is something that could be handled in a follow-up PR.

Reply:
Definitely something we could do! There's more places in our workflows where using custom actions or reusable workflows could be useful. Let's not wait for these improvements to not postpone the merge. I will refactor the workflow in the future.